src/GoogleAuthenticator.php

```diff
@@ -86,6 +86,21 @@ public function checkCode($secret, $code): bool
         return $result > 0;
     }
 
+    public function setCodePeriod(int $period):bool
+    {
+        if ($period >= 1 && $period <= 86400)) {
+            $this->codePeriod = $period;
+            return true;
```

Comment:
A setter with a response value? This looks wrong

Reply:
This allows you to test if the value was set.  eg

```php
if (!auth->setCodePeriod(300000)) {
}
```

The idea being the caller can decide what they do with the result since the only reason it would not be set is the value is outside of the predefined range.  If you would prefer to throw an exception, let me know which one.

Reply:
It's also used in the test case.

Reply:
> This allows you to test if the value was set. eg
> 
> ```
> if (!auth->setCodePeriod(300000)) {
> }
> ```
> 
> The idea being the caller can decide what they do with the result since the only reason it would not be set is the value is outside of the predefined range. If you would prefer to throw an exception, let me know which one.

If there is a problem and you can't set the value, you should throw an exception.

Reply:
Since I'm not used to using the test cases and I just ripped an existing one, how would I go testing for the exception and which exception would you want to throw?  If I was to guess, I would say http://php.net/manual/en/class.invalidargumentexception.php